spec/features/availability_spec.rb

```diff
@@ -82,6 +82,7 @@
   describe 'Archival Material:' do
     it 'has a \'Request Material\' link so it can be requested through Aeon', js: true do
       visit '/catalog/1836205'
+      click_button('View More')
```

Comment:
Is this necessary? Box 1 the first one in the first 4 listed...

Reply:
The reason I added that click is to make sure that the holding we are testing here would always be in the visible css. With being not 💯 sure, I think this is the spec that is failing time to time depending on the order of holdings listed, so I thought adding a click to view more would work around that.

Reply:
Ok, cool. Thanks.